pages/calculator/[slug].js

```diff
@@ -32,10 +37,13 @@ import {
 } from '../../utils/sanity.client.ts';
 
 export default function CalculatorSlugRoute({ page, calculatorConfig }) {
+  const router = useRouter();
   const [open, setOpen] = useState(false);
+  const [share, setShare] = useState(false);
+  const [copied, setCopied] = useState(false);
```

Comment:
Can we rename this to something more descriptive like `shareLinkCopied`.

Reply:
Of course